docs/docs-content/troubleshooting/cluster-deployment.md

```diff
@@ -94,6 +95,31 @@ Common reasons for why a service may fail are:
 
 6. Check stdout for errors. You can also open a support ticket. Visit our [support page](http://support.spectrocloud.io/).
 
+## Deployment Violates Pod Security
+In cluster profiles running Kubernetes 1.25 or later, some packs contain Pods that need elevated privileges during Pod creation. 
+When you try to deploy a cluster with such packs, you get an error message that looks like the following:
+
+```
+Error creating: pods <name of pod> is forbidden: violates PodSecurity "baseline:v<k8s version>": non-default capabilities …
+```
+
+### Debug Steps
+
+You can change the Pod Security Standards of the namepace where the Pod is being created to address this issue. 
+
```

Comment:
You can find the style we use for basic login to Palette [Update a Cluster Profile].(https://docs.spectrocloud.com/profiles/cluster-profiles/modify-cluster-profiles/update-cluster-profile).

![CleanShot 2023-10-24 at 16 15 16](https://github.com/spectrocloud/librarium/assets/117382432/91bb8aef-9d57-446d-8684-738af6f5378a)